raiden/transfer/channel.py

```diff
@@ -2774,11 +2797,12 @@ def sanity_check(channel_state: NettingChannelState) -> None:
     our_balance = get_balance(our_state, partner_state)
     partner_balance = get_balance(partner_state, our_state)
 
-    msg = (
-        "The balance can never be negative, that would be equivalent to a loan or a double spend."
-    )
-    assert our_balance >= 0, msg
-    assert partner_balance >= 0, msg
+    # msg = (
```

Comment:
YELP!

Reply:
What do we do about those?

Reply:
If we did this check after we read all claims, this would work right? In any case, at least a comment why we break this check at the moment would be nice.